### ISSUE TITLE
ERROR 001369 Failed to create the service

### ISSUE THREAD

**Issue by jingxuehan0707** [Esri/network-analysis-services#4]
We are using StreetMap Premium 2015 Release 3. The GP tool created four sd file successfully, but it failed when ArcGIS for Server tries to start the service. From the GP tool, it threw ERROR 001369 Failed to create the service.

In Server Manager, NetworkAnalysis.MapServer, NetworkAnalysisUtilities.GPServer were started successfully. But when we tried to started NetworkAnalysis.GPServer, and NetworkAnalysisSync.GPServer, it gave error in Server Manager, com.esri.arcgis.discovery.servicelib.AGSException

Just wonder if there any similar issue. I have feeling that the ArcGIS for Server might be corrupted.

Thanks.
[log.txt](https://github.com/Esri/network-analysis-services/files/283107/log.txt)

**Comment by deelesh** [comment]
Looking at log.txt, I do not think there is anything wrong with your ArcGIS Server.  I have a few questions.
1.  Is the server running ArcGIS for Server Linux?
2. Are there additional messages in the ArcGIS Server logs related to NetworkAnalysis.MapServer service that you mention started successfully?
3. Did you [install the license](https://github.com/Esri/network-analysis-services/wiki/DataSetup#data-setup-on-gis-server) for Streetmap premium data on the server machine and restart ArcGIS Server service after the data license was installed?

**Comment by jingxuehan0707** [comment]
Hi Deelesh,

Thanks for reply. Here is my answer to your question.
1. The ArcGIS for Server is Linux, the OS is Red Hat Enterprise Linux Server release 6.6.
2. Actually, I never checked log in of NetworkAnalysis.MapServer since it started successful. I did checked the DEBUG log for NetworkAnalysis.GPServer, I didn't see WARNING and SEVERE message. I will go ahead to stop and start the NetworkAnalysis.MapServer again to get the log.
3. I did go through the process of installing the license, and I restarted the Server after that.

**Comment by jingxuehan0707** [comment]
Hi Deelesh,

I stopped and started the NetworkAnalysis.MapServer and get the log. Here is the log file.
[NetworkAnalysis_MapServer.xlsx](https://github.com/Esri/network-analysis-services/files/287340/NetworkAnalysis_MapServer.xlsx)

**Comment by deelesh** [comment]
The logs tell me that NetworkAnalysis.MapServer was able to successfully connect to the licensed street map premium data. So the problem with the GP services is not related to data license.

Can you set the ArcGIS Server log level to DEBUG and just publish the NetworkAnalysisSync.sd from ArcGIS Server Manager? if the service fails to publish, please share the logs.

**Comment by jingxuehan0707** [comment]
Hi Deelesh,

Thanks for your help so far. I will go ahead to get log file for you.

**Comment by jingxuehan0707** [comment]
Hi Deelesh,

On May 31, I tried to publish the NetworkAnalysisSync.sd, and it failed right way, and I got the log from Server Manager. I attached here as 
[Publishing_service_0531.xlsx](https://github.com/Esri/network-analysis-services/files/305159/Publishing_service_0531.xlsx)
It looks like the token expired for that sd file. Not sure how the token work for that PublishingUtility python tool. I signed in to Portal every time I published.

Then, I tried to redo the whole process again, starting from scratch. It will publish NetworkAnalysis.MapServer, and NetworkAnalysisUtilities.GPServer OK. But it looks like the python tool failed right after 60 minutes, and I got the token related message in Server Manager again. Here is the python tool box log. 
[PublishRoutingServices0603.txt](https://github.com/Esri/network-analysis-services/files/305169/PublishRoutingServices0603.txt)

My plan is to change my token lifespan in Server manager from 60 minutes to 180 minutes and see if it helps.

Thanks.
